src/main/frontend/src/routes/persistence.svelte

```diff
@@ -28,16 +28,15 @@
 		if(storageType === "JOURNAL") {
 			$projectionsTypes = [
 				{name: 'Not Applicable', value: 'NONE'},
-				{name: 'Event Based', value: 'EVENT_BASED'},
+				{name: 'Event-based', value: 'EVENT_BASED'},
 			];
-			if(projections === 'OPERATION_BASED') {
-				projections = "NONE";
-			}
+			useCQRS = true;
+			projections = "EVENT_BASED";
```

Comment:
if you select `JOURNAL` then update useCQRS to false or projections  something other than 'EVENT_BASED'
After that if you refresh the page, useCQRS is true and projections is 'EVENT_BASED'

Reply:
how would we go around that?
we would need to somehow go around the first execution of `changedStorageType`.
materialify's on:change also triggers on first change, but that would typically be the easiest solution.